dist/pom.xml

```diff
@@ -1684,6 +1688,10 @@
                                                 dest="${project.build.directory}/parallel-world/${spark.version.classifier}"
                                         >
                                         </unzip>
+                                        <exec executable="${project.basedir}/scripts/binary-dedupe.sh"
```

Comment:
why are we deduping on individual, it only contains a single spark version anyway?

Reply:
top level classes are duplicated
```find  dist/target/parallel-world/ -name \*.class | xargs sha1sum | sort -k1,1 | awk '{print $2 " " $1}' | uniq -f 1 -d
dist/target/parallel-world/org/openucx/jucx/ucp/UcpListener.class 055d141a3d00e9503946e93f24e09ef46dc5bf98
dist/target/parallel-world/org/openucx/jucx/ucp/UcpEndpointErrorHandler.class 065e86ee52ca0a287d0e23d03727f4d1e6219930
dist/target/parallel-world/org/openucx/jucx/ucp/UcpListenerConnectionHandler.class 0a512cc7021c09173797dceb501fe38acfe95ddd
...
```